bartop-ui/src/apis/base.js

```diff
@@ -1,4 +1,4 @@
-import { errors } from '../strings';
+import strings from '../strings';
```

Comment:
Why throw in this extra layer? Can you not use object notation when exporting a default object? If you're only using the errors sub-object from strings, I feel like it is more efficient style to `import { errors }`

Reply:
Agreed but yeah, you can't use `destructuring` on `import`s so you gotta `import` the root object

Reply:
Sad!